django/django.md

```diff
@@ -0,0 +1,19 @@
+OVERVEIW<br>
+Django,free and opensource web framework,which allows us to create secure and well maintained websites.Primary goal of django is to make the development of complex database driven websites easy.
+In order to write web applications in python directly basic requirements include:<br>
+1.Web server gateway interface.<br>
+2.Routing<br>
+3.SQL Interface<br>
+4.Template<br><br>
+
+On the other hand Django includes all of the above mention features hence it becomes easy for one to write web application.One can focus on the designing part and writing of the web application without taking care of the above features.Various sites that use django includes pinterest Public Broadcasting Service, Instagram, Mozilla, The Washington Times, Disqus, Bitbucket and Nextdoor.<br><br>
+
+Django is fast,fully loaded,secure,scalable(can handle large increase in users,traffic on various sites) and versatile.Despite of the above stated advantages django is bad for large applications and it comes with tons of perbuilt features,which may or maynot be used,will consume server resources.<br><br>
+
+RESOURCES TO LEARN DJANGO:
+Django Girls Tutorial[https://tutorial.djangogirls.org/en/index.html]<br>
+How To Tango With Django 1.7[http://www.tangowithdjango.com/book17/]<br>
+Mastering Django for Django 1.8 [https://mastering-django.thinkific.com/]<br>
+
+DJANGO INSTALLATION GUIDE[https://docs.djangoproject.com/en/2.0/intro/install/]

```

Comment:
4. Here, can you please make "DJANGO INSTALLATION" as a hyperlink, instead of providing a link.
You can use, Github cheatsheet for that.